kernel-hal/src/bare/arch/riscv/drivers.rs

```diff
@@ -93,20 +99,5 @@ pub(super) fn intc_init() -> DeviceResult {
     irq.unmask(ScauseIntCode::SupervisorSoft as _)?;
     irq.unmask(ScauseIntCode::SupervisorTimer as _)?;
 
-    #[cfg(feature = "graphic")]
```

Comment:
graphic function is useful

Reply:
graphic_init() will be call in drivers::init(), I removed it from drivers::intc_init() because graphic module has nothing to do with `intc`.